searx/static/themes/simple/src/js/main/keyboard.js

```diff
@@ -43,6 +43,29 @@ searxng.ready(function () {
     }
   });
 
+  searxng.on('.result', 'keydown', function (e) {
+    var resultElement = getResultElement(e.target);
+    if (isImageResult(resultElement)) {
+      var matchKey = true;
+      switch (e.key){
+      case 'ArrowLeft':
+        searxng.selectPrevious();
+        break;
+      case 'ArrowRight':
+        searxng.selectPrevious();
```

Comment:
Typo, should be `searxng.selectNext();` :)

Reply:
thanks, solved.